Spanner/src/Connection/Grpc.php

```diff
@@ -166,10 +183,13 @@ public function __construct(array $config = [])
             $grpcConfig['apiEndpoint'] = $config['apiEndpoint'];
         }
 
-        $this->spannerClient = isset($config['gapicSpannerClient'])
-            ? $config['gapicSpannerClient']
-            : $this->constructGapic(SpannerClient::class, $grpcConfig);
+        $this->enableResourceCaching = isset($config['enableCaching']) && $config['enableCaching'];
```

Comment:
it seems that `enableCaching` will always be set
Perhaps
```php
        $this->enableResourceCaching = $config['enableCaching'];
```
is enough.

Reply:
Such a record always requires the key 'enableCaching' in the $config array. If it is not there, then in the case when the php error_reporting setting displays an error line, you can see a message about the absence of the index.
![config11](https://user-images.githubusercontent.com/15671241/71081922-358b6300-21a1-11ea-904e-758767f7e4ae.jpg)
I made the proposed changes, as you said that there will always be some value in this array key.